exporter/clickhouselogsexporter/migrations/000001_init_db.up.sql

```diff
@@ -0,0 +1,59 @@
+CREATE TABLE IF NOT EXISTS logs (
+	timestamp UInt64 CODEC(Delta, ZSTD(1)),
```

Comment:
Since we are allowing user to parse the timestamp and we don't want to enforce any default timezone we are using Uint64 but will check how it will affect ttl.